Week3/homework/step2-1.js

```diff
@@ -1,15 +1,13 @@
 'use strict';
 
-function foo(func) {
-  // What to do here?
-  // Replace this comment and the next line with your code
-  console.log(func);
-}
-
 function bar() {
   console.log('Hello, I am bar!');
 }
 
+function foo(func) {
+  return func();
```

Comment:
Nice. No need to `return` though - seeing as you're not using the return value of calling `foo`

Reply:
Ok

Reply:
we do not return value, only console.log()